locale/templates/settings.ftl

```diff
@@ -208,6 +209,15 @@ input-password-hide-aria = Hide password from screen.
 input-password-show-aria = Show password as plain text. Your password will be visible on screen.
 
 
+## Linked Accounts section
+
+la-heading = Linked Accounts
+la-description = You have authorized access to the following accounts.
+la-unlink-button = Unlink
+la-unlink-account-button = Unlink
+la-unlink-heading = Unlink from third party account
+la-unlink-content = Are you sure you want to unlink your { -brand-google } account? Unlinking your account does not automatically sign you out of those services, to do that you will need to manually sign out from the Connected services section.
```

Comment:
The punctuation needs to change. Comma should be a period: 
`...sign you out of those services, to do that you will....` -> `...sign you out of those services. To do that, you will....` .